### ISSUE TITLE
implements device_state_attributes after update Home Assistant 2021.12.0

### ISSUE THREAD

**Issue by serlinGi** [itchannel/fordpass-ha#137]
hi,
after update HA december I received this warning in log:
``` Logger: homeassistant.helpers.entity
Source: helpers/entity.py:549
First occurred: 12:07:31 AM (28 occurrences)
Last logged: 12:07:36 AM

Entity sensor.fordpass_remotestartstatus_2 (<class 'custom_components.fordpass.sensor.CarSensor'>) implements device_state_attributes. Please report it to the custom component author.
Entity device_tracker.fordpass_tracker (<class 'custom_components.fordpass.device_tracker.CarTracker'>) implements device_state_attributes. Please report it to the custom component author.
Entity device_tracker.fordpass_tracker_2 (<class 'custom_components.fordpass.device_tracker.CarTracker'>) implements device_state_attributes. Please report it to the custom component author.
Entity sensor.fordpass_odometer_2 (<class 'custom_components.fordpass.sensor.CarSensor'>) implements device_state_attributes. Please report it to the custom component author.
Entity sensor.fordpass_odometer (<class 'custom_components.fordpass.sensor.CarSensor'>) implements device_state_attributes. Please report it to the custom component author.
```
Thank's for all your efforts

**Comment by itchannel** [comment]
Thanks for the heads up I'll get and update to the new schema this weekend.

**Comment by itchannel** [comment]
Fixed in #138 👍
